content/api/commands/selectfile.md

```diff
@@ -0,0 +1,281 @@
+---
+title: selectFile
+---
+
+Selects a file or files in an HTML5 input element or simulates dragging a file
+or files into the browser.
+
+## Syntax
+
+```javascript
+.selectFile(file)
+.selectFile(file, options)
+.selectFile([file1, file2, ...])
+.selectFile([file1, file2, ...], options)
+```
+
+### Usage
+
+**<Icon name="check-circle" color="green"></Icon> Correct Usage**
+
+```javascript
+cy.get('input[type=file]').selectFile('file.json')
+cy.get('input[type=file]').selectFile(['file.json', 'file2.json'])
+
+cy.get('input[type=file]').selectFile({
+  contents: Cypress.Buffer.from('file contents'),
+  fileName: 'file.txt',
+  lastModified: Date.now(),
+})
+
+cy.get('input[type=file]').selectFile('file.json', { action: 'drag-drop' })
+cy.document().selectFile('file.json', { action: 'drag-drop' })
+```
+
+**<Icon name="exclamation-triangle" color="red"></Icon> Incorrect Usage**
+
+```javascript
+// Errors, cannot be chained off 'cy'
+cy.selectFile('file.json')
+
+// Will attempt to find a file called 'file contents'
+// on disk, probably not what you intended
+cy.get('input[type=file]').selectFile('file contents')
+```
+
+### Arguments
+
+**<Icon name="angle-right"></Icon> file** **_(String, Array, Object or
+Cypress.Buffer)_**
+
+Either a single file, or an array of files. A file can be:
+
+- A path to a file within the project root (the directory that contains the
+  default Cypress configuration file). Eg: `'path/to/file.json'`
+- `@alias` - An alias of any type, previously stored using `.as()`. Eg:
+  `'@alias'`
+- A [`Cypress.Buffer()`](/api/utilities/buffer) containing binary data, such as
```

Comment:
I don't think we want to show `Cypress.Buffer` being called as a function. How about this?

```suggestion
- A [`Cypress.Buffer`](/api/utilities/buffer) instance containing binary data, such as
```